languages/ruby/exercises/concept/floating-point-numbers/savings_account_test.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
```

Comment:
```suggestion

```

The two strings can be garbage collected, and are not in danger of being mutilated.